doc/release-notes/0.10/0.10.md

```diff
@@ -0,0 +1,187 @@
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.10.0 release contains the following known issues and limitations:
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1">
+<thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Platform</th>
+<th valign="bottom">Impact</th>
+<th valign="bottom">Workaround</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/378">#378</a></td>
+<td valign="top">DDR support</td>
+<td valign="top">AIX</td>
+<td valign="top">Inability to diagnose problems with the VM, garbage collector, or JIT.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/54">#54</a></td>
+<td valign="top">Signal handling</td>
+<td valign="top">All platforms</td>
+<td valign="top">Currently, shutdown signals (<code>SIGINT</code>, <code>SIGHUP</code> and <code>SIGTERM</code>) and <code>SIGCONT</code> are fully supported on Unix platforms (pLinux, zLinux, xLinux, AIX and zOS). Full support for shutdown signals is pending on Windows; signal handlers cannot be registered by using <code>sun.misc.Signal</code> or <code>dk.internal.misc.Signal</code>. Also, support for other POSIX signals is pending. See <code>SunMiscSignalTest.java</code> for the list of signals that need to be supported.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/479">#479</a></td>
+<td valign="top">Non-compressed references VM builds not available</td>
+<td valign="top">AIX, Windows, Linux on Z, Linux on Power</td>
+<td valign="top">Heaps larger than 57GB are not supported.</td>
+<td valign="top">Manual builds are possible by following our <a href="https://github.com/eclipse/openj9/blob/master/buildenv/Build_Instructions_V8.md">detailed build instructions</a>.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/897">#897</a></td>
+<td valign="top">Javadoc not building some OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation is missing in the <code>build/&lt;platform&gt;/docs</code> directory.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/898">#898</a></td>
+<td valign="top">Javadoc building some unsupported OpenJ9 extensions.</td>
+<td valign="top">All platforms</td>
+<td valign="top">Some API documentation in <code>build/&lt;platform&gt;/docs</code> is not supported with OpenJ9.</td>
+<td valign="top">None</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/ibmruntimes/openj9-openjdk-jdk8/issues/17">#17</a></td>
+<td valign="top">Builds contain extra shared libraries.</td>
```

Comment:
This only applies to Java 8 (and 10) now. The issue is resolved for Java 11 (and 0.10.0 is a Java 11 only release).

Reply:
Fixed